### PR TITLE
{bio}[goolf-1.7.20] SAMtools 1.2 (Review)

### PR DESCRIPTION
**Pull request by npklein**

### CODE REVIEW

**Comment by npklein** [comment]
Test report by @npklein
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2697 v2 @ 2.70GHz, Python 2.7.9
See https://gist.github.com/d2b302a85da58aa46004 for a full test report.

**Comment by npklein** [comment]
Test report by @npklein
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2697 v2 @ 2.70GHz, Python 2.7.9
See https://gist.github.com/24643b06c4b4d1ba0fdf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f6f271e2e40d4a4edc10 for a full test report.

**Comment by boegel** [comment]
@rjeschmi: please review?

**Comment by rjeschmi** [comment]
This should be built against zlib 1.2.8 I think

@fgeorgatos ?

**Comment by boegel** [comment]
@npklein: any specific reason for picking zlib 1.2.7?

**Comment by npklein** [comment]
@boegel To keep it equal to dependencies in SAMtools-1.2-goolf-1.4.10.eb

**Comment by rjeschmi** [comment]
@npklein there is a few other versions of samtools-1.2 around. The foss-2015a pulls 1.2.8 and goolf-1.7.20 already has a zlib 1.2.8 available.

I think if you just switch the zlib dep and remove the older zlib easyconfig (I don't think we will really need to support 1.2.7 is goolf-1.7.20) it should be good to go.

File: easybuild/easyconfigs/z/zlib/zlib-1.2.7-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'zlib'
+version = '1.2.7'
```
Comment (boegel) [inline easybuild/easyconfigs/z/zlib/zlib-1.2.7-goolf-1.7.20.eb]: @npklein: do we need to keep this one around?

If you remove it, also adjust the PR title please.

Comment (boegel) [inline easybuild/easyconfigs/z/zlib/zlib-1.2.7-goolf-1.7.20.eb]: ping?

**Comment by fgeorgatos** [comment]
@rjeschmi: 
yes, it is as you described, zlib/1.2.8 is probably preferred in the goolf/1.7.20 domain (by no means to block anybody, just stating a preference)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4855512d6e45c3ece337 for a full test report.

**Comment by boegel** [comment]
going in, thanks @npklein

**Pull request merged**